tutorials/MD/data/md.mdp

```diff
@@ -1,314 +1,45 @@
-init-lambda              = 0
-init-lambda-state        = -1
-delta-lambda             = 0
-nstdhdl                  = 50
-fep-lambdas              = 
-mass-lambdas             = 
-coul-lambdas             = 
-vdw-lambdas              = 
-bonded-lambdas           = 
-restraint-lambdas        = 
-temperature-lambdas      = 
-calc-lambda-neighbors    = 1
-init-lambda-weights      = 
-dhdl-print-energy        = no
-sc-alpha                 = 0
-sc-power                 = 0
-sc-r-power               = 6
-sc-sigma                 = 0.3
-sc-coul                  = no
-separate-dhdl-file       = yes
-dhdl-derivatives         = yes
-dh_hist_size             = 0
-dh_hist_spacing          = 0.1
-
-; Non-equilibrium MD stuff
-acc-grps                 = 
-accelerate               = 
-freezegrps               = 
-freezedim                = 
-cos-acceleration         = 0
-deform                   = 
-
-; simulated tempering variables
-simulated-tempering      = no
-simulated-tempering-scaling = geometric
-sim-temp-low             = 300
-sim-temp-high            = 300
-
-; Ion/water position swapping for computational electrophysiology setups
-; Swap positions along direction: no, X, Y, Z
-swapcoords               = no
-adress                   = no
-
-; User defined thingies
-user1-grps               = 
-user2-grps               = 
-userint1                 = 0
-userint2                 = 0
-userint3                 = 0
-userint4                 = 0
-userreal1                = 0
-userreal2                = 0
-userreal3                = 0
-userreal4                = 0
-; Electric fields
-; Format for electric-field-x, etc. is: four real variables:
-; amplitude (V/nm), frequency omega (1/ps), time for the pulse peak (ps),
-; and sigma (ps) width of the pulse. Omega = 0 means static field,
-; sigma = 0 means no pulse, leaving the field to be a cosine function.
-electric-field-x         = 0 0 0 0
-electric-field-y         = 0 0 0 0
-electric-field-z         = 0 0 0 0
+; Run parameters
+integrator              = md        ; leap-frog integrator
+nsteps                  = 500000    ; 2 * 500000 = 1000 ps (1 ns)
+dt                      = 0.002     ; 2 fs
+; Output control
+nstxout                 = 0         ; suppress bulky .trr file by specifying 
+nstvout                 = 0         ; 0 for output frequency of nstxout,
+nstfout                 = 0         ; nstvout, and nstfout
+nstenergy               = 5000      ; save energies every 10.0 ps
+nstlog                  = 5000      ; update log file every 10.0 ps
+nstxout-compressed      = 5000      ; save compressed coordinates every 10.0 ps
+compressed-x-grps       = System    ; save the whole system
+; Bond parameters
+continuation            = no        ; Not continuing from previous simulation 
+constraint_algorithm    = lincs     ; holonomic constraints 
+constraints             = h-bonds   ; bonds involving H are constrained
+lincs_iter              = 1         ; accuracy of LINCS
```

Comment:
I believe the modern gmx way to handle these variables is to use a `-` instead of a `_`, it doesn't have any impact on 2020, but it could do on future versions. It might be safer to change these here where possible to avoid having to think about it in future releases?

I've added comment suggested changes so as to speed up the process.

Reply:
_ changed to - in .mdp files